internal/buildscripts/packaging/choco/splunk-otel-collector/tools/LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+﻿VERIFICATION

```

Comment:
LICENSE.txt should be similar to https://github.com/signalfx/signalfx-agent/blob/main/scripts/windows/choco/signalfx-agent/tools/LICENSE.txt, but the link to the license file should be `https://raw.githubusercontent.com/signalfx/splunk-otel-collector/main/LICENSE`.

Reply:
`LICENSE.txt` updated.